### ISSUE TITLE
Demo: Bootstrap in the marketplace

### ISSUE THREAD

**Issue by paddyow** [Kademi/kademi-dev#3307]
http://rewardtest1.admin.kademi.us/marketPlace/bootstrap/

Should we make this more visible that it is the latest bootstrap component that a customer will need, and give it an image?

I can create an image for this.

**Comment by bradmac** [comment]
Definitely needs a tidy up.

**Comment by bradmac** [comment]
The default theme is installed when a webite is created. But we do want to integrate theme selection with the marketplace, so a user just installs the theme app, and it is selected as the current theme

**Comment by bradmac** [comment]
New UI allows searching market place by apps or themes

![image](https://user-images.githubusercontent.com/826458/31111002-5882f888-a867-11e7-8dac-800eb1232836.png)

**Comment by bradmac** [comment]
And website manager has updated UI on the theme tab:
![image](https://user-images.githubusercontent.com/826458/31111020-7ac3bc16-a867-11e7-8566-0363c1ff4009.png)

By going to the marketplace from here, and selecting a theme it is automatically installed and selected as the website theme

**Comment by Uafinder** [comment]
YEah, it works. But theme checker seems to show bootstrap theme anytime...
![image](https://user-images.githubusercontent.com/1945230/31332343-b720463a-acee-11e7-95a2-09219952c601.png)
![image](https://user-images.githubusercontent.com/1945230/31332356-c56fc940-acee-11e7-8d5e-115e61054973.png)

http://vlad-test10.admin.kademi-ci.co/websites/vlad10-blog/version1/#panel_theme-tab

**Comment by bradmac** [comment]
The theme checker is rendering with the paper theme correctly

You can see the original theme here - https://bootswatch.com/paper/

Note that this is a bootstrap theme, so the theme checker still mentions bootstrap

**Comment by bradmac** [comment]
From original theme:
![image](https://user-images.githubusercontent.com/826458/31355784-1c005da2-ad98-11e7-8983-4d8c1be5d621.png)

From theme checker:
![image](https://user-images.githubusercontent.com/826458/31355807-30d8235e-ad98-11e7-834b-10663c4f5def.png)

**Comment by Uafinder** [comment]
Not quite understand...
I want to check my new theme and click Theme checker... but it shows me my standard theme.
For example here
http://vlad-test31.admin.kademi-ci.co/websites/vlad31-learn/version1/#panel_theme-tab

Cyborg theme original
![image](https://user-images.githubusercontent.com/1945230/32379341-f38ed4c6-c0b5-11e7-979e-0547981a817e.png)
Our
![image](https://user-images.githubusercontent.com/1945230/32379356-014aeb86-c0b6-11e7-81ee-313843e26671.png)
![image](https://user-images.githubusercontent.com/1945230/32379445-3ac91b6c-c0b6-11e7-964a-39e96fb33d21.png)
Where i'm wrong?

**Comment by bradmac** [comment]
Is working now:
![image](https://user-images.githubusercontent.com/826458/32419166-a8a44d7a-c2da-11e7-958a-9db35ab8be26.png)

Note that i've just restarted the CI server, so its possible this is a server caching issue. Can you please attempt to reproduce?

**Comment by Uafinder** [comment]
Without any problems
1. I've create a new website
2. Check it's theme, it's by default. Click to check it (theme checker)
3. Then go to themes and install cosmo theme.
4. Check theme checker again, but it shows default k-theme.
